pallets/creditcoin/src/lib.rs

```diff
@@ -1439,5 +1443,20 @@ pub mod pallet {
 			CollectCoinsContract::<T>::put(contract);
 			Ok(())
 		}
+
+		#[transactional]
+		#[pallet::weight(<T as Config>::WeightInfo::del_authority())]
+		pub fn del_authority(
```

Comment:
```suggestion
 		pub fn delete_authority(
```

Let's not abbreviate if not necessary.

Reply:
agree.